### DISCUSSION TITLE
Base url clarification

### DISCUSSION THREAD

**Discussion by johnwc** [elsa-workflows/elsa-core#3145] [Q&A] [answered]
The BaseUrl setting that we need to add/update in appsettings, when  we deploy to a production environment, what should this be? Should it be localhost:port, IP:port of the workflow server node, or FQDN:port of the registered DNS for the dashboard node?

 - https://localhost:7294
 - https://10.42.15.120:7294
 - https://workflows.domain.com

**Comment by sfmskywalker** [comment, marked as answer]
The `BaseUrl` needs to be set to a publicly available URL at which the workflow server app is reachable. This URL is used when generating absolute URLs. For example, when generating a URL for triggering a signal.

**Reply by johnwc** [reply]
So option 3? The FQDN to the dashboard server?

**Reply by sfmskywalker** [reply]
Yes, sorry for not being clear: option 3 :)
